### DISCUSSION TITLE
Building unikraft linuxu as .so

### DISCUSSION THREAD

**Discussion by Julius1991** [unikraft/unikraft#763] [Q&A]
Dear unikraft community,

I want to build unikraft linuxu (and the application that is running on top) as a shared object to link it into another executable.
Applying -shared and -fPIC in https://github.com/unikraft/unikraft/blob/staging/Makefile.uk is not a big deal, but is anything else required (e.g. currently -fno-PIC and -no-pie are used)?

Thanks for your support,
Julius

**Comment by razvand** [comment]
@Julius1991, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
